src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
I completely understand that it's redundant, but here like in other places, it's only redundant because of the context or the operation being done. RGB alone doesn't make sense to I'd rather always qualify it with what kind of RGB it is.